src/neptune/new/internal/operation.py

```diff
@@ -197,21 +203,16 @@ def of_file(cls, value: File, attribute_path: List[str], upload_path: Path):
                 file_path=os.path.abspath(value.path),
             )
         elif value.file_type in (FileType.IN_MEMORY, FileType.STREAM):
-            tmp_file_path = cls.get_upload_path(attribute_path, value.extension, upload_path)
-            value._save(tmp_file_path)
-            operation = UploadFile(
-                path=attribute_path,
-                ext=value.extension,
-                file_path=os.path.abspath(tmp_file_path),
-                clean_after_upload=True,
-            )
+            tmp_file_name = cls.get_tmp_file_name(attribute_path, value.extension)
+            value._save(upload_path / tmp_file_name)
+            operation = UploadFile(path=attribute_path, ext=value.extension, tmp_file_name=tmp_file_name)
```

Comment:
`clean_after_upload` is missing. Is it intentional?

Reply:
Yes. Now we determine if operation should be cleaned by existence of `tmp_file_name` value.

`clean_after_upload` is deprecated flag leaved only for handling of old format operations if user invoke sync on old data.